plugins/base/src/test/kotlin/translators/DefaultPsiToDocumentableTranslatorTest.kt

```diff
@@ -1,5 +1,6 @@
 package translators
 
+import com.jetbrains.rd.util.first
```

Comment:
That's an interesting package, I actually have no idea what `rd` stands for. But it seems to be coming from one of the intellij dependencies.

To avoid possible compatibility problems in the future (can be deleted/moved/changed any day for all we know), I would change it to an stdlib alternative. 

For example, instead of doing
```kotlin
map.first().value
``` 

I think you can do
```kotlin
map.values.first()
```